### PR TITLE
Restsharp109

### PR DESCRIPTION
**Pull request by mjkkirschner**
replaces:
https://github.com/DynamoDS/Dynamo/pull/14490

we could not consume 110 in Dynamo when targeting net6.
slack thread for future reference:
https://autodesk.slack.com/archives/C01C8GGT9EY/p1697764048583789

- [x] https://github.com/DynamoDS/PackageManagerClient/pull/89 must be merged first.

### CODE REVIEW

**Review by pinzart90 — APPROVED** [review]

File: src/DynamoCore/DynamoCore.csproj
Comment (aparajit-pratap) [inline src/DynamoCore/DynamoCore.csproj:35]: Duplicates

File: src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj
Comment (aparajit-pratap) [inline src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj:18]: ⚠️

**Comment by mjkkirschner** [comment]
@aparajit-pratap thanks for catching these.

File: src/DynamoCore/DynamoCore.csproj
Comment (aparajit-pratap) [inline src/DynamoCore/DynamoCore.csproj:34]: @mjkkirschner I'm still seeing 2 versions of LibG nugets for 229 and 230?

File: src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj
Comment (aparajit-pratap) [inline src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj:17]: Same here, still seeing 2 versions of libg nuget.

Comment (mjkkirschner) [reply src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj:17]: thanks @aparajit-pratap - totally missed that as I did not touch LibG in the nuget UI so did not expect it to be affected, I will fix it.

Comment (aparajit-pratap) [reply src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj:17]: I think you might need to pull the latest master to your PR branch if you haven't already done so before making these commits.

Comment (mjkkirschner) [reply src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj:17]: thanks @aparajit-pratap I need to update to whatever the actual published version of Greg.dll ends up being so I will do that then.

**Review by aparajit-pratap — APPROVED** [review]
LGTM assuming you'll be updating to the final version of Greg soon.

**Comment by mjkkirschner** [comment]
this should now start passing, I will keep an eye on it before merging.

**Pull request merged**
